packages/health-gorilla-core/src/aoe.ts

```diff
@@ -51,24 +50,16 @@ export function getMissingRequiredQuestionnaireItems(
           'https://www.healthgorilla.com/fhir/StructureDefinition/questionnaire-requiredwhenspecimen'
         ) === true)
     ) {
-      required.push(item.linkId);
-
       const answers = answersByLinkId?.[item.linkId];
       if (answers && answers.length > 0) {
-        let hasValue = false;
         for (const answer of answers) {
           const values = arrayify(
             getTypedPropertyValue({ type: 'QuestionnaireResponseItemAnswer', value: answer }, 'value[x]')
           );
           if (values?.some((v) => v.value)) {
-            hasValue = true;
             continue;
```

Comment:
This is the unnecessary `continue` according to Biome:
<img width="867" alt="Screenshot 2025-02-22 at 3 30 58 PM" src="https://github.com/user-attachments/assets/e163c950-4a2a-4075-a046-8388e1b5153b" />